tests/foreman/cli/test_logging.py

```diff
@@ -0,0 +1,272 @@
+"""CLI tests for logging.
+
+:Requirement: Logging
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: logging
+
+:TestType: Functional
+
+:CaseImportance: Medium
+
+:Upstream: No
+"""
+import re
+from fauxfactory import gen_string
+from robottelo import (
+    manifests,
+    ssh,
+)
+from robottelo.cli.factory import (
+    make_org,
+    make_product_wait,
+    make_repository,
+)
+from robottelo.cli.subscription import Subscription
+from robottelo.ssh import upload_file
+from robottelo.test import CLITestCase
+
+
+def line_count(self, file, connection=None):
+    """Get number of lines in a file."""
+    connection = connection or ssh.get_connection()
+    result = connection.run('wc -l < {0}'.format(file),
+                            output_format='plain')
+    self.assertEqual(result.return_code, 0, "BASH command error?")
+    return result
```

Comment:
Standalone functions shouldn't take class instances as first argument. Either remove `self` and assertion, or move that function into `SimpleLoggingTestCase` and call it like: `number_of_lines = self.line_count(file, connection)`.